### PR TITLE
fix: Module initializer bug from PR2402

### PR DESCRIPTION
**Pull request by IXLLEGACYIXL**
# PR Details

[PR2402](https://github.com/stride3d/stride/pull/2402) introduced a problem which i dont know exactly what it was
xen found that nugetresolver was called too late and after swithcing the generation order a bit it started to work again

## Related Issue

[PR2402](https://github.com/stride3d/stride/pull/2402)

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] **I have built and run the editor to try this change out.**

### CODE REVIEW

File: sources/core/Stride.Core.AssemblyProcessor/AssemblyProcessorApp.cs [hunk 1]
```diff
@@ -214,15 +214,12 @@ public bool Run(ref AssemblyDefinition assemblyDefinition, ref bool readWriteSym
                 if (SerializationAssembly)
                 {
                     processors.Add(new AssemblyScanProcessor());
-                    processors.Add(new SerializationProcessor());
                 }
-
-                if (ModuleInitializer)
+                if (SerializationAssembly)
```
Comment (Jklawreszuk) [inline sources/core/Stride.Core.AssemblyProcessor/AssemblyProcessorApp.cs:218]: Question: This 'if' duplicate was intended right?

Comment (IXLLEGACYIXL) [reply sources/core/Stride.Core.AssemblyProcessor/AssemblyProcessorApp.cs:218]: this PR is just to see the difference to master where it doesnt work

**Review by Jklawreszuk — COMMENTED** [review]
I tested now and indeed seems to work

**Review by Jklawreszuk — APPROVED** [review]

File: deps/AssemblyProcessor/netstandard2.0/Stride.Core.AssemblyProcessor.pdb
Comment (NicusorN5) [inline deps/AssemblyProcessor/netstandard2.0/Stride.Core.AssemblyProcessor.pdb:1]: Is there a reason to include PDBs? Shouldn't they be generated by the compiler?

Comment (IXLLEGACYIXL) [reply deps/AssemblyProcessor/netstandard2.0/Stride.Core.AssemblyProcessor.pdb:1]: the deps.json is also included ... the gitignore isnt that good yet

**Comment by IXLLEGACYIXL** [comment]
closing in favor of #2477

**Pull request closed without merging**